libr/include/r_util/r_num.h

```diff
@@ -2,7 +2,7 @@
 #define R_NUM_H
 
 #define R_NUMCALC_STRSZ 1024
-#define r_num_abs(x) x>0?x:-x
+#define R_NUM_UNITS_BUFLEN 31 /* used in r_num_units() */
```

Comment:
imho its wrong to have such define here, because the size of the buffer constructed to do this is internal to the function call. if you want to fix this, just add (and pass) the buffer length to the function instead of asuming the caller have allocated X bytes. also 31 is just a random guess, nothing scientific at all

Reply:
I guess that `r_num_units` need at least 6 bytes: 4 bytes for number (0-1023), 1 byte for unit and 1 for NULL.

Reply:
oh I forget the decimal number, so we need at least 8 bytes.